eng/pipelines/notifications.yml

```diff
@@ -34,47 +34,55 @@ stages:
     variables:
       Organization: azure-sdk
       Project: internal
+      DotNetDevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
 
     steps:
 
-      - task: UseDotNet@2
-        displayName: 'Use .NET Core sdk $(DotNetCoreVersion)'
+      - task: DotNetCoreCLI@2
+        displayName: 'Install Notification Creator'
         inputs:
-          version: '$(DotNetCoreVersion)'
+          command: custom
+          custom: 'tool'
+          arguments: 'install --global --add-source "$(DotNetDevOpsFeed)" --version "1.0.0-dev.20211018.1" "Azure.Sdk.Tools.NotificationConfiguration"'
```

Comment:
We should share this version in some common place so that we can keep it consistent and updated across the repo.